docs/api/connectivity/bluetooth/Optimizing_for_performance.md

```diff
@@ -118,16 +118,18 @@ will only have to turn on the radio when the expected packet is due.
 
 ## Increasing throughput
 
+The following approaches can help to increase throughput, but you might need to consider whether power consumption will be adversely affected in your application.
```

Comment:
@pan- @paul-szczepanek-arm - are you ok with this extra text? (added to avoid the empty section after heading and to sign post a little)

Reply:
Lgtm